temp_example.json

```diff
@@ -0,0 +1,129 @@
+{
+    "pre_mapped": {
+        "id": "ga4gh:VA.FLe4-pSUs7vjdVtVD4TmUNL4JhrBbqTd",
+        "type": "Allele",
+        "extensions": [
+            {
+                "name": "vrs_ref_allele_seq",
+                "value": "Y"
+            }
+        ],
+        "digest": "FLe4-pSUs7vjdVtVD4TmUNL4JhrBbqTd",
+        "location": {
+            "type": "SequenceLocation",
+            "digest": "DCfpyPamywb6xZ_YuqheLIUUna9idFdK",
+            "sequenceReference": {
+                "type": "SequenceReference",
+                "refgetAccession": "SQ.PyX9IDu95_tYLg1Jz9JpW5xpQkwn6bpB"
+            },
+            "start": 169,
+            "end": 170
+        },
+        "state": {
+            "type": "LiteralSequenceExpression",
+            "sequence": "G"
+        }
+    },
+    "post_mapped": {
+        "id": "ga4gh:VA.rKyjzmt0czvrVFeRsvCxH-aE4GSoMzUS",
+        "type": "Allele",
+        "extensions": [
+            {
+                "name": "vrs_ref_allele_seq",
+                "value": "Y"
+            }
+        ],
+        "digest": "rKyjzmt0czvrVFeRsvCxH-aE4GSoMzUS",
+        "expressions": [
+            {
+                "syntax": "hgvs.p",
+                "value": "NP_938033.1:p.Tyr439Gly"
+            }
+        ],
+        "location": {
+            "type": "SequenceLocation",
+            "digest": "F_PJZIrk2lQaj2CLaS-TbsWdeJjwAsCu",
+            "sequenceReference": {
+                "type": "SequenceReference",
+                "refgetAccession": "SQ.uJDQo_HaTNFL2-0-6K5dVzVcweigexye"
+            },
+            "start": 438,
+            "end": 439
+        },
+        "state": {
+            "type": "LiteralSequenceExpression",
+            "sequence": "G"
+        }
+    },
+    "pre_mapped_1_3": {
```

Comment:
previously we had provided VRS/VRSATILE variations as the "main" mapping product. with this PR, we don't any more, but can optionally include VariationDescriptors under these `pre_mapped_1_3` and `post_mapped_1_3` keys for backwards-compatibility reasons. Should we
a) include the whole VOD, as we had been (lets us include `vrs_ref_allele_seq` and a post-mapped `expression`
b) just include the variation
c) just include a VA ID